admin/config-ui/class-configuration-page.php

```diff
@@ -104,8 +104,7 @@ public function show_wizard() {
 				printf(
 					/* translators: %s expands to Yoast SEO. */
 					__( '%s &rsaquo; Configuration Wizard', 'wordpress-seo' ),
-					'Yoast SEO'
-				);
+					'Yoast SEO' );
```

Comment:
We've discussed this over Slack. There are alternative solutions which might suit the project better like:
```php
// Prepare the vars you need beforehand.
$a = sprintf(...);
?>
<title><?php echo $a; ?></title>
```